Assets/TanksDemo/Assets/Scripts/UI/ControlRebindingUI.cs

```diff
@@ -89,6 +89,7 @@ void RemapButtonClicked(string name, int bindingIndex = 0)
     {
         m_Button.enabled = false;
         m_Text.text = "Press button/stick for " + name;
+        m_RebindOperation?.Dispose();
```

Comment:
Unrelated but just realized that after the recent change to `ToHumanReadableString`, line 131 should be

```
        m_Text.text = InputControlPath.ToHumanReadableString(m_Action.bindings[0].effectivePath,
            InputControlPath.HumandReadableStringOptions.OmitDevice);
```